### ISSUE TITLE
Why not show the latest releases in github "releases"?

### ISSUE THREAD

**Issue by NormandoHall** [sveltejs/kit#9059]
### Describe the bug

Today I was updated SK to 1.6.0 and the latest release shows in github releases is 1.5.5. It is very useful to know the changes.

![image](https://user-images.githubusercontent.com/2943127/219127490-2402e065-0aa8-46c2-aadb-7284bac0ae47.png)

Thanks

### Reproduction

No reproduction available

### System Info

```Shell
System:
    OS: Linux 3.10 CentOS Linux 7 (Core)
    CPU: (4) x64 Intel Xeon Processor (Skylake, IBRS)
    Memory: 577.67 MB / 15.51 GB
    Container: Yes
    Shell: 4.2.46 - /bin/bash
  Binaries:
    Node: 16.15.0 - /usr/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 9.2.0 - /usr/bin/npm
  npmPackages:
    @sveltejs/adapter-node: ^1.2.0 => 1.2.0 
    @sveltejs/kit: ^1.5.6 => 1.5.6 
    svelte: ^3.55.1 => 3.55.1 
    vite: ^4.1.1 => 4.1.1
```

### Severity

annoyance

**Comment by Conduitry** [comment]
Short version: The next versions we release should hopefully be tagged. We shall see when the next release happens. I don't know that we want to retroactively manually tag the couple of versions that we missed, though.

Long version: At one point, the automatic release job started publishing a couple of internal packages that we didn't actually want published to npm. I had manually unpublished those from npm, but the release job continued to try to republish them. Since npm prevents republishing the same version of a package that had previously been unpublished, these publishes failed. Apparently, the way the GitHub action doing the publishing and tagging is written, if any of the npm publishes fail, it doesn't push the tags to Git or create a release in GitHub, even for the npm publishes that succeeded. Since the last releases, we have merged #9051, which should address the issue where we were trying to republish version that we had previously unpublished from npm, which should in turn start creating tags in Git and releases in GitHub.
